### ISSUE TITLE
Cron task link creation fails

### ISSUE THREAD

**Issue by jpalomaki** [ansible/ansible#2957]
Cron job registration:

 `cron: name=foo user=jpalomaki job="echo foo" minute="*/5" state=present`

fails with:

`msg: ln: creating hard link /etc/cron.d/jpalomaki' => /tmp/tmpK0hiMV': Invalid cross-device link`

and is probably caused by this (/tmp being on a different LVM partition):

```
[root@machine ~]# df -h
Filesystem            Size  Used Avail Use% Mounted on
/dev/mapper/vg_00-root 43G  7.3G   34G  18% /
/dev/mapper/vg_00-tmp 279M   11M  255M   4% /tmp
```

**Comment by bcoca** [comment]
what version are you running?

this should have been fixed in current devel (1.2)

**Comment by jpalomaki** [comment]
We're running 1.1.

**Comment by mpdehaan** [comment]
Yep, this is fixed in 1.2

1.2 is due for release within the first half of June, though many people are running the development branch now.

We are unlikely to release a 1.1 backport as we have not encountered any other major issues at this time, and we're so close to to the 1.2 date.

**Comment by jpalomaki** [comment]
Ok, thanks for your efforts
